### PR TITLE
add rest api and node documentation for custom notifications

### PR DESCRIPTION
**Pull request by ofoucherot**
### Description

- [x] Add REST API doc for trigger inbox notification endpoint.
- [ ] (ISSUE) Update REST API doc of the response schema for get inbox notification endpoint.
- [x] Update liveblocks-node documentation.

### CODE REVIEW

File: docs/pages/api-reference/liveblocks-node.mdx [hunk 1]
```diff
@@ -1585,6 +1570,57 @@ await liveblocks.deleteRoomNotificationSettings({
...
+
+#### Liveblocks.triggerInboxNotification [#trigger-inbox-notification] [@badge=Beta]
+
+Triggers an inbox notification based on an activity that happened in your
+system. This is a wrapper around the
+[Trigger Inbox Notification API](/docs/api-reference/rest-api-endpoints#post-inbox-notifications-trigger).
+
+`kind` must start with a `$`.
+
+`activityData` can have `string`, `number`, or `boolean`.
```
Comment (marcbouchenoire) [inline docs/pages/api-reference/liveblocks-node.mdx:1599]: I would also put this with "`kind` must start with a `$`." above, not as a new paragraph.

```suggestion
Properties of `activityData` can have `string`, `number`, or `boolean` values.
```

File: docs/pages/api-reference/liveblocks-node.mdx [hunk 2]
```diff
@@ -1585,6 +1570,57 @@ await liveblocks.deleteRoomNotificationSettings({
...
+
+```ts
+await liveblocks.triggerInboxNotification({
+  // The ID of the user that will receive the inbox notification.
+  userId: "steven@example.com",
+
+  // The type of notification, must start with a $.
```
Comment (marcbouchenoire) [inline docs/pages/api-reference/liveblocks-node.mdx:1606]: I've been talking about this as "custom kind" in other places, is that a good way to describe this?

```suggestion
  // The custom kind of notification, which must start with a $.
```

Comment (CTNicholas) [reply docs/pages/api-reference/liveblocks-node.mdx]: I think it sounds better if we go with "Custom notification kind" than "custom kind of notification"

**Review by CTNicholas — APPROVED** [review]
Thanks for starting this @ofoucherot! Made a couple of small changes 👍

File: docs/pages/api-reference/liveblocks-node.mdx (hunk 2 above)
Comment (marcbouchenoire) [reply docs/pages/api-reference/liveblocks-node.mdx]: Agree, I'll adjust my code comments if necessary.

**Pull request merged**